### ISSUE TITLE
Markdown table formatting has been broken 

### ISSUE THREAD

**Issue by dulajra** [h2oai/wave#2339]
#### Wave SDK Version, OS

Wave 1.1.0

#### Actual behavior

![Screenshot 2024-06-04 at 16 01 43](https://github.com/h2oai/wave/assets/12434380/155315a7-b995-45b6-a91e-fa39072fbae1)

#### Expected behavior

![Screenshot 2024-06-04 at 16 02 26](https://github.com/h2oai/wave/assets/12434380/7ef2918e-2cdf-449c-bc12-d0b368d3761b)

#### Steps To Reproduce

```
from h2o_wave import site, ui, main


def make_markdown_row(values):
    return f"| {' | '.join([str(x) for x in values])} |"


def make_markdown_table(fields, rows):
    return '\n'.join([
        make_markdown_row(fields),
        make_markdown_row('-' * len(fields)),
        '\n'.join([make_markdown_row(row) for row in rows]),
    ])


page = site['/demo']

fields = ['Column 1', 'Column 2']
rows = [['Row 1 Column 1', 'Row 1 Column 2'], ['Row 2 Column 1', 'Row 2 Column 2']]

v = page.add('example', ui.form_card(
    box='1 1 4 5',
    items=[
        ui.text(make_markdown_table(
            fields=fields,
            rows=rows
        ),),
    ],
))

page.save()

```

**Comment by dulajra** [comment]
@mturoci Can you please check? Assign option is disable for me. cc: @jakubhava

**Comment by jakubhava** [comment]
We are not looking necessarily for a fix @mturoci , if you could please suggest a workaround if any, that would be great as well

**Comment by mturoci** [comment]
Seems like a build problem of some sort:

<img width="799" alt="image" src="https://github.com/h2oai/wave/assets/64769322/9e7eaad3-0c72-4298-8889-8908c0b60375">

@marek-mihok any ideas?

**Comment by marek-mihok** [comment]
> @marek-mihok any ideas?

@mturoci, vite build had problem with this `React.useEffect(() => { import('./markdown.css') }, [])`, also if [awaited](https://vitejs.dev/guide/features#dynamic-import). 

Fixed by #2341
